### DISCUSSION TITLE
【功能建议】建议增加除帧脚本

### DISCUSSION THREAD

**Discussion by echoIIImk2** [hooke007/mpv_PlayKit#341] [回收]
一些老片会存在将24帧压成30帧的问题，这时候无论是用补帧软件还是电视的补帧功能都失效，需要先除帧。似乎是有除帧脚本的，但是小白不容易搞定，建议增加一个一键除帧的功能。

**Comment by hooke007** [comment]
https://github.com/hooke007/MPV_lazy/wiki/3_K7sfunc#ivtc_std

仓库就有示例
https://github.com/hooke007/MPV_lazy/blob/main/portable_config/vs/ETC_IVTC_STD.vpy
而且也捆绑在 vsLite/vsMega 包里

不知道怎么用就照抄input呗
这里不存在技能障碍，只是不熟悉文档的问题
https://github.com/hooke007/MPV_lazy/blob/179f177ef418e8e4f0fec8477b6bd5a581eb9ed6/portable_config/input_list.conf#L39-L75
